client/src/components/mapping/TabColumnPicking/StaticValueForm/index.tsx

```diff
@@ -138,14 +178,96 @@ const StaticValueForm = ({ attribute }: Props) => {
       await createStaticInput({
         variables: {
           attributeId,
-          staticValue
+          staticValue: customSystem
+            ? `http://terminology.arkhn.org/${selectedSource!.id}/${
+                selectedResource!.id
+              }`
+            : staticValue
         }
       });
     } catch (e) {
       console.log(e);
     }
   };
 
+  const renderAddStaticValueButton = (): React.ReactElement => (
+    <Button
+      disabled={
+        (!customSystem && staticValue.length === 0) ||
+        (customSystem && selectedResource === undefined)
+      }
+      icon={'add'}
+      loading={creatingStaticInput}
+      onClick={onAddStaticValue}
+    />
+  );
+
+  const handleSourceSelect = (source: Source): void => {
+    setSelectedSource(source);
+    setSelectedResource(undefined);
+  };
+  const handleResourceSelect = (resource: Resource): void => {
+    setSelectedResource(resource);
+  };
+
+  const renderCustomSystemInput = (): React.ReactElement => (
```

Comment:
I think that should be an other component (alongside with selectedSource and selectedResource)

Reply:
In the `selects` folder?
It's not really a select but I agree that this could be somewhere else

Reply:
no, in the StaticValueForm folder directly. something like `<IdentifierSystemInput>`

Reply:
Done